admin/filters/docLink.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+function before_filter_docLink($data, $folder) {
+    return preg_replace_callback('/:doc:`(.*?) <(.*?)>`/', function ($matches) {
```

Comment:
`:doc:` has two patterns.

```
1) :doc:`uploaded files </libraries/uploaded_files>`
2) :doc:`./services`
```

Reply:
Ok, thanks - I wasn't scanning all the docs yet.